src/components/styles/PageSection.styled.ts

```diff
@@ -13,34 +13,51 @@ const fadeInUp = keyframes`
     }
 `
 
-export const MainContainer = styled.div<{ hasImage: boolean }>`
-  margin: 75px 0;
+export const MainContainer = styled.div<{ hasImage: boolean; mayNeedReverse?: boolean }>`
+  margin-top: 20px;
+  margin-bottom: 75px;
   display: flex;
   padding: 0 50px 0 0;
   justify-content: space-between;
   background-color: black;
   animation-duration: 1s;
   animation-name: ${fadeInUp};
+
+  @media screen and (max-width: 800px) {
+    flex-direction: ${(props) => (props.mayNeedReverse ? 'column-reverse' : 'column')};
+    align-items: center;
+  }
 `
 
-export const ImageContainer = styled.div<{ image: string; textPosition: string }>`
-  height: 400px;
-  width: 400px;
+export const ImageContainer = styled.div<{ image: string; textPosition?: 'left' | 'right'; events?: boolean }>`
+  max-width: 350px;
+  min-height: 300px;
+  width: ${(props) => (props.events ? '20%' : '50%')};
   background-size: contain;
   background-repeat: no-repeat;
   background-position: center;
-  ${(props) => (props.textPosition === 'right' ? 'margin-left: 100px' : 'margin-right: 20px')};
   ${(props) => (props.image === '' ? `background-image: url(${Null});` : `background-image: url(${props.image});`)}
```

Comment:
would taking the similar texts out of the equation be better? like `background-image: ${props...};`

Reply:
it will remove the image when the screen width < 800px